src/main/scala/outwatch/dom/VDomModifier.scala

```diff
@@ -9,13 +9,31 @@ import snabbdom.{DataObject, VNodeProxy, hFunction}
 import scala.scalajs.js
 import collection.breakOut
 
-sealed trait VDomModifier_ extends Any
+sealed trait Modifier extends Any
 
-case class CompositeVDomModifier(modifiers: Seq[VDomModifier]) extends VDomModifier_
 
-case class Emitter(eventType: String, trigger: Event => Unit) extends VDomModifier_
+// Modifiers
 
-sealed trait Property extends VDomModifier_
+sealed trait Property extends Modifier
+
+final case class Emitter(eventType: String, trigger: Event => Unit) extends Modifier
+
+private[outwatch] final case class AttributeStreamReceiver(attribute: String, attributeStream: Observable[Attribute]) extends Modifier
```

Comment:
Why are these private now?

Reply:
Because it should only be used from within the outwatch package. Is that not the case?